docs/1.0. EDID-Mapping.md

```diff
@@ -0,0 +1,220 @@
+
+## Use of Normative Language
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](#rfc2119-ref).
+
+## Definitions
+
+_See also the [NMOS Glossary](https://github.com/AMWA-TV/nmos/wiki/Glossary), and definitions within RFCs._
+
+
+### IPMX 
+  The IPMX (Internet Protocol for Media Experience) is a set of open standards and specifications to enable the carriage of compressed and uncompressed video, audio and data over IP networks. It includes provisions for control, copy protection, connection management and security. Includes use of SMPTE and NMOS standards.
+
+
+### IPMX Controller
+  An IPMX Controller is a type of NMOS Controller analogous to an NMOS Broadcast Controller. The IPMX Controller is at the application layer of the NMOS Architecuture and makes use of IPMX APIs including AMWA NMOS APIs. Further information can be found in [EDID-CM-ARCH](#edid-cm-arch-ref).  
+
+### EDID 
+  Extended Display Identification Data (EDID) is a metadata format for display devices to describe their capabilities to a video source (e.g. graphics card or set-top box). The data format is defined by a standard published by the Video Electronics Standards Association (VESA).  For purposes of this BCP, EDID definitions are taken from the E-EDID version 1.4 [E-EDID](#edid-ref).
+
+## Mapping from EDID to NMOS Resources
+
+This section is broken down into several subsections that describe recommendations for mapping parts of a target display device's EDID to NMOS Resources.  The following subsections are provided:
+
+- Mapping of EDID to IS-04 Receiver Caps 
+   - Video Mappings from EDID to IS-04 Receiver Caps
+   - Audio Mappings from EDID to IS-04 Receiver Caps
+- Mapping of EDID to IS-EDID-CM
+   - EDID to IS-EDID-CM Destination Resource
+   - EDID to Essential Media Capabilities
+   - EDID to Raw EDID Format
+
+### Constuction of IS-04 Receiver Caps
+
+This section describes how a display device's capabilities (video and audio) as described by its EDID is mapped by an implementation to the NMOS device's [IS-04](#is-04-ref) Receiver Capabilities.  
+
+#### Video and AUdio Mappings
+
+The following refers to the NMOS Receiver Capabilities Best Current Practices [BCP-004-01](#bcp-004-01-ref). Paremeter values used are taken from the NMOS Capabilities Parameter Register [NMOS-CAP-REG](#nmos-cap-reg-ref)
+
+##### Video Mappings
+
+In order that a NMOS device that provides IS-04 Receivers can work with EDID type video devices the device:
+
+- MUST create one IS-04 Video Receiver for each of the displays associated with the IS-04 Device
```

Comment:
This is a point for discussion, I'd love to understand the motivation behind this

Reply:
I agree with Chris. The relationship between displays and Devices is not defined, and gateway devices may consume a single stream and pass it to multiple displays.